src/test/java/org/reaktivity/nukleus/oauth/internal/control/ControllerIT.java

```diff
@@ -248,8 +249,13 @@ public void shouldResolveWithIssuerAndAudience() throws Exception
     {
         k3po.start();
 
+        JwtClaims claims = new JwtClaims();
+        claims.setClaim("iss", "test issuer");
+        claims.setClaim("aud", "testAudience");
+        String payload = claims.toJson();
```

Comment:
This JSON would be `{ "iss": "test issuer", "aud": "testAudience" }` but we want the more readable `{ "issuer": "test issuer", "audience": "testAudience" }` since these are security constraints that might contain non-claims going forward, such as required freshness to enforce `iat` dynamically based on the current time.